### DISCUSSION TITLE
mqtt client was disconnect,but on rabbitmq dashboard the connection is still connect and it cant be disconnect forever.

### DISCUSSION THREAD

**Discussion by springmvcGitHub** [rabbitmq/rabbitmq-server#7890] [Questions]
mqtt client was disconnect,but on rabbitmq dashboard the connection is still connect and it cant be disconnect forever.
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/42227896/231714887-de8fb5fd-d5de-44bd-bf4e-aef49a1a1d61.png">

And i get some logs from rabbitmq:
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/42227896/231716305-b60eeaea-e143-408d-8d5f-c4dc84839e42.png">

The end,i disconnect the connection manually on dashboard.

**Comment by michaelklishin** [comment]
How can we reproduce this behavior? Can you put together an executable way to reproduce in a public repo here on GitHub? Using `3.12.0-beta.6`.

**Comment by michaelklishin** [comment]
A member of our team tried to reproduce this with `3.12.0-beta.6` and could not. Please put together an executable way to reproduce or at least a detailed set of steps that would reproduce this issue with `3.12.0-beta.6`.

**Comment by springmvcGitHub** [comment]
Actually,it no problem in my own computer use 3.12.0-beta.2 or 3.12.0-beta.6,it has this problem when i deploy it on Azure Iot Edge(3.12.0-beta.2 or 3.12.0-beta.6 both has this problem),could you try it on Azure Iot Edge?
